config/http.conf

```diff
@@ -84,6 +84,9 @@ http {
   add_header    X-Proxy-Backend-Response-Time   $upstream_response_time;
   add_header    X-Proxy-Request-Time            $request_time;
 
+  # Observability headers
+  add_header    X-Trace-ID    $otel_trace_id;
```

Comment:
The advantage of having a proxy everywhere is we can set headers like this. Figured it would be nice to have this available in the response (which also means it's in the browser too)!